CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Use python 3.8 in Github actions. ([#659](https://github.com/jina-ai/finetuner/pull/659))
```

Comment:
Move this to 0.7 section? it will be included in the release

Reply:
ah i'll move it later